packages/cli/src/models/TestHelper.ts

```diff
@@ -1,14 +1,14 @@
 import ControllerFor from '../models/network/ControllerFor';
 import ZosNetworkFile from '../models/files/ZosNetworkFile';
-import { SimpleProject, AppProject } from 'zos-lib';
+import { ProxyAdminProject, AppProject } from 'zos-lib';
 
 /**
  * Initializes a zOS application testing and deploying it to the test network,
  * along with dependencies (if specified)
  * @param txParams optional txParams (from, gas, gasPrice) to use on every transaction
  * @param networkFile optional `ZosNetworkFile` object to use, instead of zos.test.json
  */
-export default async function(txParams: any = {}, networkFile?: ZosNetworkFile): Promise<SimpleProject | AppProject> {
+export default async function(txParams: any = {}, networkFile?: ZosNetworkFile): Promise<ProxyAdminProject | AppProject> {
```

Comment:
Good catch. It makes much more sense to use the ProxyAdmin for tests rather than the Simple one.